collectors/ebpf.plugin/README.md

```diff
@@ -39,815 +35,826 @@ The following two features from the Linux kernel are used by Netdata to run eBPF
-to the function `mark_buffer_dirty`.
+You can also enable the following eBPF programs:
 
-#### Page cache ratio
+- `cachestat`: Netdata's eBPF data collector creates charts about the memory page cache. When the integration with
+  [`apps.plugin`](/collectors/apps.plugin/README.md) is enabled, this collector creates charts for the whole host _and_
+  for each application.
+- `dcstat` : This eBPF program creates charts that show information about file access using directory cache. It appends
+  `kprobes` for `lookup_fast()` and `d_lookup()` to identify if files are inside directory cache, outside and files are
+  not found.
+- `disk` : This eBPF program creates charts that show information about disk latency independent of filesystem.
+- `filesystem` : This eBPF program creates charts that show information about some filesystem latency.
+- `swap` : This eBPF program creates charts that show information about swap access.
+- `mdflush`: This eBPF program creates charts that show information about
+  multi-device software flushes.
 
-When the processor needs to read or write in a specific memory address, it checks for a corresponding entry in the page cache.
-If the processor hits a page cache (`page cache hit`), it reads the entry from the cache. If there is no entry (`page cache miss`),
- the kernel allocates a new entry and copies data from the disk. Netdata calculates the percentage of accessed files that are cached on
- memory. The ratio is calculated counting the accessed cached pages
- (without counting [dirty pages](#dirty-pages) and pages added because of read misses) divided by total access without dirty pages.
+### Configuring eBPF threads
 
-\_\_________<ins>Number of accessed cached pages</ins>__________
-Number of total accessed pages - dirty pages - missed pages
+You can configure each thread of the eBPF data collector. @thiagoftsm Can you give some details why/ when you should configure a thread? 
 
-The chart `cachestat_ratio` shows how processes are accessing page cache. In a normal scenario, we expect values around
-100%, which means that the majority of the work on the machine is processed in memory. To calculate the ratio, Netdata
-attaches `kprobes` for kernel functions:
+To configure an eBPF thread:
 
-- `add_to_page_cache_lru`: Page addition.
-- `mark_page_accessed`: Access to cache.
-- `account_page_dirtied`: Dirty (modified) pages.
-- `mark_buffer_dirty`: Writes to page cache.
+1. Navigate to the [Netdata config directory](/docs/configure/nodes.md#the-netdata-config-directory).
+    ```bash
+    cd /etc/netdata
+    ```
+2. Use the [`edit-config`](/docs/configure/nodes.md#use-edit-config-to-edit-configuration-files) script to edit a thread configuration file. The following configuration files are available:
 
-#### Page cache misses
+   - `network.conf`: Configuration for the [`network viewer` thread](#network-configuration). This config file overwrites the global options and also
```

Comment:
Please, remove the word `viewer`, because this is still a work in progress. We have now a `network thread`.